level_02.md

```diff
@@ -89,7 +89,13 @@ Create `.eslintrc.json`:
 }
 ```
 
-Add a lint script to run `eslint`, and then `npm run lint` (because `lint` isn't one of the standard commands, you need the extra `run`).
+Add a lint script to `packages.json` to run `eslint`:
+
+```
+"lint": "eslint *.js"
+```
+
+and then `npm run lint` (because `lint` isn't one of the standard commands, you need the extra `run`).
```

Comment:
Just a note that the need for `run` is a little more interesting than `lint` not being a standard command. `npm` has a bunch of lifecycle scripts that get called by various commands. Some of them happen to have the same name as the commands (like `build` and `test`). Don't know if that's an important distinction to make at this point, though.

Reply:
Well, I managed this far without really knowing the specific reason, so probably not (though an interesting TIL). Perhaps a footnote?

Reply:
I first discovered it when `npm` ran some of my scripts unexpectedly because I'd given them a magic name. Not a fun one to debug. https://docs.npmjs.com/cli/v6/using-npm/scripts